EarlGrey.xcodeproj/project.pbxproj

```diff
@@ -1018,6 +1038,8 @@
 				OTHER_CFLAGS = (
 					"-fobjc-arc",
 					"-fobjc-arc-exceptions",
+					"-DNS_BLOCK_ASSERTIONS=1",
```

Comment:
If the framework does require `NS_BLOCK_ASSERTIONS` to be undefined, is there any reason why build settings couldn't explicitly undefine it? I guess I'm just wondering what the hesitation is with adding the flag.